coda/config/settings/base.py

```diff
@@ -74,7 +74,8 @@ def get_secret(setting, secrets=secrets):
         'HOST': get_secret("DB_HOST"),
         'PORT': get_secret("DB_PORT"),
         'OPTIONS': {
-            'init_command': 'SET default_storage_engine=MyISAM'
+            'init_command': 'SET default_storage_engine=MyISAM',
+            'sql_mode': 'traditional',
```

Comment:
Noting that we discussed not using 'traditional', since we are using a non-transactional storage engine.